.github/workflows/end_to_end_test.yaml

```diff
@@ -12,8 +12,14 @@ jobs:
     steps:
       - name: Echo hello world
         run: echo "hello world"
+      - name: File permission for /usr/local/bin
+        run: ls -ld /usr/local/bin | grep drwxrwxrwx
```

Comment:
Should we just do something in the directory that requires the permission?

Reply:
Thanks. This is added.